public/scripts/fight/FightScene.js

```diff
@@ -20,39 +21,40 @@ export class FightScene extends Scene {
         text: 'TIME TO FIGHT! coming later...\nE/SPACE TO RETURN',
       }),
       skillA1Tooltip: new Tooltip({
-        x: 300,
-        y: 400,
-        width: 100,
+        name: 'skill',
+        shape_id: '0',
+        x: data.stage.width()/6,
+        y: data.stage.height() - 100,
+        width: 300,
         height: 50,
-        text: 'A1 Z',
+        text: `${this.inventory.equipped[0].name}`,
       }),
       skillA2Tooltip: new Tooltip({
-        x: 450,
-        y: 400,
-        width: 100,
+        name: 'skill',
+        shape_id: '1',
+        x: data.stage.width()/6 + 300,
+        y: data.stage.height() - 100,
+        width: 300,
         height: 50,
-        text: 'A2 X',
+        text: `${this.inventory.equipped[1].name}`,
       }),
       skillA3Tooltip: new Tooltip({
-        x: 600,
-        y: 400,
-        width: 100,
+        name: 'skill',
+        shape_id: '2',
+        x: data.stage.width()/6,
+        y: data.stage.height() - 200,
+        width: 300,
         height: 50,
-        text: 'A3 C',
+        text: `${this.inventory.equipped[2].name}`,
       }),
       skillA4Tooltip: new Tooltip({
-        x: 750,
-        y: 400,
-        width: 100,
+        name: 'skill',
+        shape_id: '3',
+        x: data.stage.width()/6 + 300,
+        y: data.stage.height() - 200,
+        width: 300,
         height: 50,
-        text: 'A4 V',
-      }),
```

Comment:
We have the info for each Item stored as an `Item` instance, could we put the `item.type` and `item.dmg` property in the text of the boxes so we can tell what pressing each skill does?